src/components/navBar/navBar.js

```diff
@@ -498,7 +504,9 @@ function MdNavItem($mdAria, $$rAF, $mdUtil, $window) {
           });
         }
 
-        $mdAria.expectWithText(element, 'aria-label');
+        if (!ctrl.navItemAriaLabel) {
+          $mdAria.expectWithText(element, 'aria-label');
```

Comment:
This is still checking for the `aria-label` on the `li` element. Replacing `element` with `navButton` here seems to properly check the `md-button._md-nav-button` element.